llm/.env.example

```diff
@@ -0,0 +1,11 @@
+# WANDB
+WANDB_API_KEY=1234567890
+WANDB_ENTITY=user_name
```

Comment:
Another pitfall I think we should mention: when I log into wandb, I use my email `jhiemstra@wisc.edu`, but this `WANDB_ENTITY` expects only the `jhiemstra` part (as I guess wandb differentiates email and username). It took me awhile to figure out that this was why the job was failing for me, but admittedly I haven't used weights and biases in awhile.

Reply:
Added link to profile to check `WANDB_ENTITY` / user name